vng_api_common/extensions/serializer_extensions.py

```diff
@@ -0,0 +1,78 @@
+from drf_spectacular.extensions import OpenApiSerializerExtension
+from drf_spectacular.openapi import AutoSchema
+from drf_spectacular.plumbing import ResolvedComponent
+
+from vng_api_common.utils import underscore_to_camel
+
+
+class GegevensGroepFieldExtension(OpenApiSerializerExtension):
+    target_class = "vng_api_common.serializers.GegevensGroepSerializer"
+    match_subclasses = True
+
+    def map_serializer(self, auto_schema: AutoSchema, direction):
+        schema = auto_schema._map_serializer(
+            self.target, direction, bypass_extensions=True
+        )
+
+        del schema["description"]
+
+        return schema
+
+
+class PolymorphicSerializerExtension(OpenApiSerializerExtension):
+    target_class = "vng_api_common.polymorphism.PolymorphicSerializer"
+    match_subclasses = True
+
+    def map_serializer(self, auto_schema, direction):
+        if not getattr(self.target, "discriminator", None):
+            raise AttributeError(
+                "'PolymorphicSerializer' derived serializers need to have 'discriminator' set"
+            )
+
+        discriminator = self.target.discriminator
+
+        # resolve component with base path
+        base_schema = auto_schema._map_serializer(
+            self.target, direction, bypass_extensions=True
+        )
+        base_name = f"Base_{self.target.__class__.__name__}"
+        base_component = ResolvedComponent(
+            name=base_name,
```

Comment:
It's possible that request and response schemas for Polymorphic serializer can be different, so if different schemas for request and response are turned off, it would be nice to allow it. You can take a look at the [implementation](https://github.com/open-zaak/open-zaak/blob/cb59636ed63b3ecd61a935c3a84a6521b51c7a55/src/openzaak/utils/oas_extensions/polymorphic.py#L26) in OZ for it